api/src/main/java/org/apache/gravitino/model/ModelVersionChange.java

```diff
@@ -71,6 +79,26 @@ static ModelVersionChange updateUri(String newUri) {
     return new ModelVersionChange.UpdateUri(newUri);
   }
 
+  /**
+   * Create a ModelVersionChange for updating the aliases of a model version.
+   *
+   * @param newAlias The new aliases to be set for the model version.
+   * @return A new ModelVersionChange instance for updating the alias of a model version.
+   */
+  static ModelVersionChange updateAlias(String... newAlias) {
```

Comment:
I would suggest changing the interface to something like:

```
static ModelVersionChange updateAliases(String[] aliasesToAdd, aliasesToRemove)
```

So this one API can support both adding and deleting aliases. What do you think?

Reply:
`aliasesToAdd ` can be null or empty, which means no new aliases to add.
`aliasesToDelete ` can be null or empty, which means no new aliases to delete.
if both are empty, then nothing happens.